ios/Classes/TiBarcodeViewController.m

```diff
@@ -89,26 +94,107 @@ - (UIInterfaceOrientation)preferredInterfaceOrientationForPresentation
 - (BOOL)shouldAutorotate
 {
   [super shouldAutorotate];
-    
+
   if (_preventRotation) {
     return NO;
   }
-    
+
   return YES;
 }
 
 - (void)handleDeviceRotation:(NSNotification *)notification
 {
   if (_showRectangle) {
     CGRect rect = _overlayView.cropRect;
-    [_scanner setScanRect:rect];
+    // [_scanner setScanRect:rect];
```

Comment:
I think commented line should be removed (Or did you commented it to revisit ? ).